### ISSUE TITLE
Getting started (for Linux + Android) contains several bugs

### ISSUE THREAD

**Issue by saevarb** [facebook/react-native#10702]
### Description

After following your getting started guide for Linux, on two different
machines(my friend and I doing the same things on each our laptop), we ran into
a couple of issues with the guide.

1. The guide says nothing about the Android SDK that react-native expects.
The guide mentions Android 7.0, but following the steps to create a project results in maven build files that refer to version 6.0. This will result in an error when trying to perform `react-native run-android`.

2. The guide to setting up the `$PATH` variable is incomplete.
When running `react-native run-android`, it needs to use `adb` to talk to the android device/emulator(AFAIU). It returns an error `sh: adb: command not found`, but seemingly continues running regardless. I didn't let it finish running so I'm not sure what happens if it continues without the `adb` binary available, but I'm assuming it won't work? The `adb` binary is located in `$ANDROID_HOME/platform_tools`

### Reproduction

Follow the guide. Install the SDK with android studio as is recommended. Set up the paths as recommended.

Then `react-native run-android` will complain as below but continue running.
```
/bin/sh: adb: command not found
```

At some point, since you have installed the newest SDK as per the guide, you will get(or similar)
```
failed to find target with hash string android-23:
```
which can be fixed by installing version 6 of the SDK.

Note: I can fix the guide for the `$PATH`, but I'm not sure what is the right way wrt. the SDK version.
Can I ask react-native to use the newer SDK, or am I simply supposed to install version 6?

### Additional Information

* React Native version:
react-native-cli: 1.2.0
react-native: 0.36.1
* Platform: Android
* Operating System: Arch Linux

**Comment by felipetozato** [comment]
I found a way to set a newer built-tool version
Inside your project folder, open the file android > app > src > build.gradle
Scrolling a little bit you can find the build configuration:
```
android {
    compileSdkVersion 25
    buildToolsVersion "25"

    defaultConfig {
        applicationId "com.awesomeproject"
        minSdkVersion 16
        targetSdkVersion 25
        versionCode 1
        versionName "1.0"
        ndk {
            abiFilters "armeabi-v7a", "x86"
        }
    }
...
```
But it would be great if during the project creation it sets by default to the latest version available.

**Comment by saevarb** [comment]
Yes, I found this file as well, but I didn't dare change it manually since I was unsure whether react-native only worked with the version it had chosen. Also, I'm unsure whether

1. This is a bug in react-native that should be fixed, which means the documentation doesn't need updating(well, maybe temporarily).
2. The documentation should tell people to install the older version instead.
3. Or whether the documentation should instruct people to update the build file manually(this sounds like the worst option)

**Comment by hramos** [comment]
I hadn't seen this issue yet, but I think some of these have now been addressed by PR #10710. One question: is is `platform-tools` or `platform_tools` on Linux? On macOS, the `adb`is in `platform-tools`.

In the PR mentioned above, I bumped the required version back down to 6.0 as there may be issues with Android 7.0 that have yet to be ironed out.

**Comment by hramos** [comment]
Closing now that #10710 has landed. Please open a new issue if the doc needs to be updated further.
